annotation-processor/src/main/java/com/bloxbean/cardano/client/plutus/annotation/processor/blueprint/BlueprintAnnotationProcessor.java

```diff
@@ -81,6 +81,7 @@ public boolean process(Set<? extends TypeElement> annotations, RoundEnvironment
             try {
                 plutusContractBlueprint = PlutusBlueprintLoader.loadBlueprint(blueprintFile);
             } catch (Exception e) {
+                e.printStackTrace();
```

Comment:
is this necessary?

Reply:
Let's keep this for now till we find a better way to notify error during compilation.